SIPS/sip-68.md

```diff
@@ -0,0 +1,97 @@
+---
+sip: 68
+title: Minor enhancements to StakingRewards.sol
+status: Proposed
+author: Clinton Ennis (@hav-noms)
+discussions-to: <https://discordapp.com/invite/AEdUHzt>
+
+created: 2020-07-06
+---
+
+<!--You can leave these HTML comments in your merged SIP and delete the visible duplicate text guides, they will not appear and may be helpful to refer to if you edit it again. This is the suggested template for new SIPs. Note that an SIP number will be assigned by an editor. When opening a pull request to submit your SIP, please use an abbreviated title in the filename, `sip-draft_title_abbrev.md`. The title should be 44 characters or less.-->
+
+## Simple Summary
+
+<!--"If you can't explain it simply, you don't understand it well enough." Simply describe the outcome the proposed changes intends to achieve. This should be non-technical and accessible to a casual community member.-->
+
+The StakingRewards contracts for Liquidity mining need some minor enhancements.
```

Comment:
[np] I've been using the convention of surrounding contract names with backticks for clarity